chordspy/chords_serial.py

```
@@ -0,0 +1,280 @@
+        global start_time, last_ten_minute_time, total_packet_count, cumulative_packet_count
+        current_time = time.time()
+        start_time = current_time            # Session start time
+        last_ten_minute_time = current_time  # 10-minute interval start time
+        total_packet_count = 0               # Counter for packets in current second
+        cumulative_packet_count = 0          # Counter for all packets
+
+    def log_one_second_data(self):
+        """
+        Log data for one second intervals and displays: Number of packets received in the last second, Number of missing samples (if any)
+        """
+        global total_packet_count, samples_per_second, missing_samples
```

Flake8 (7.2.0)

[error] `global missing_samples` is unused: name is never assigned in scope

(F824)